### PR TITLE
Added status api service for basic health checking

### LINKED ISSUE #31: Add status API
Plain text response on /status with 200 OK. Just to check if the service and web server are running.

### PR DESCRIPTION
**Pull request by woutslakhorst**
It returns  an empty body with statusCode 200 OK on /status

Fixes #31 

Signed-off-by: Wout Slakhorst <wout.slakhorst@nedap.com>

### CODE REVIEW

File: docs/pages/monitoring/nuts-consent-bridge.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+A status endpoint is provided to check if the service is running and if the web server has been started.
+The endpoint is available over http so it can be used by a wide range of health checking services.
+It does not provide any information on the individual engines running as part of the executable.
+The main goal of the service is to give a YES/NO answer for if the service is running?
+
+.. code-block::
+
+    GET /status
+
+It'll return an empty response with a 200 status code.
```
Comment (reinkrul) [inline docs/pages/monitoring/nuts-consent-bridge.rst]: Although no-one will (probably) care, shouldn't it be 201 (no content) when the response is empty?

Comment (woutslakhorst) [reply docs/pages/monitoring/nuts-consent-bridge.rst]: Should be yes, But some status checkers are quite picky and use 200 as default...

Comment (woutslakhorst) [reply docs/pages/monitoring/nuts-consent-bridge.rst]: Hmmm better to add the "OK" string in the body then everything is up to standards again

Comment (reinkrul) [reply docs/pages/monitoring/nuts-consent-bridge.rst]: Better still, conform to the Spring Boot health indicator response:
```
{
    "status" : "UP"
}
```

**Pull request merged**